CHANGELOG.rst

```diff
@@ -23,6 +23,9 @@ in development
   tooz for coordination. (improvement)
 * Allow user to specify which branch of ``st2tests`` repository to use by passing ``-b`` option to
   ``st2-self-check`` script. (improvement)
+* Allow use of webhook payload in value context.
```

Comment:
Please provide more context (now it's not really clear to what this applies to).

E.g. "Add webhook payload to the context which is available to Jinja variable inside rule criteria section" or similar.